content/articles/jira-web.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Jira Web"
+date: "2020-07-19"
+type: "post"
+authors: ["dittren"]
+tags: [jira]
+lede: "Jira net"
+shortlede: "Jira net"
+poster: "poster-jira-web.jpg"
+thumbnail: "thumbnail-jira-web.jpg"
+socmediaimg: "socmediaimg-jira-web.jpg"
+hiliteimg: "poster-jira-web.jpg"
+poster_sourceurl: "https://unsplash.com/photos/J7rRzjba-kY"
+poster_source: "Ariana Suárez on Unsplash"
+bookendanimal: "cat"
+---
+
+One of my first tasks at the CTL was to look into a more efficient way for the
+Media Team to get the information they needed when an Event requires photo
+and/or video. After some brainstorming, I wanted to approach the problem in a
+way that would be the least distruptive to the current workflow. I settled on
+creating a customized issue for photo/event purposes.
+
+This decision began my deep dive into Jira. Before Jira, I had some experience
+with project management tools like Trello and Asana but didn’t know much about
+the administrator tasks. At first, I imagined the work would mostly consist of
+creating custom fields to place on a template but I came to find that there was
+a lot more to it. Jira is a much more complex tool than I was expecting and I
+found myself in a bit of a web.
+
+The creation of a custom issue involved multiple steps. To accomplish this, yes
+you do need to create each individula custom field you would want to use in
+your issue. But it also involves the creation of an Issue Type, which is then
+attached to an Issue Type Scheme. From there, you must create a Screen and
```

Comment:
In Susan's comment, you may want to link to the Jira's documentation on Issue Type Scheme